backend/databases/qdrant.py

```
@@ -0,0 +1,105 @@
+                        "location": location,
+                        "storage_box": storage_box,
+                        "brand": brand,
+                        "size": size,
+                        "condition": condition,
+                        "image_data": image_data,  # Include base64 image for visual search results
+                        "username": username  # SECURITY VULNERABILITY - Storing username without verification
+                    }
+                )
+            ]
+        )
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)